implementation.js

```diff
@@ -0,0 +1,55 @@
+'use strict';
+var ES = require('es-abstract/es6');
+
+/*! https://mths.be/array-from v0.2.0 by @mathias */
+module.exports = function from(arrayLike) {
+	var defineProperty = (function() {
```

Comment:
all this defineProperty code can be removed if `define-properties` is added as a dep? I feel like unless we're going for "all inline, zero deps", we should use external deps whenever possible.